peps/pep-0751.rst

```diff
@@ -0,0 +1,901 @@
+PEP: 751
+Title: A file format to list Python dependencies for installation reproducibility
+Author: Brett Cannon <brett@python.org>
```

Comment:
Paul as PEP-Delegate?
```suggestion
Author: Brett Cannon <brett@python.org>
PEP-Delegate: Paul Moore <p.f.moore@gmail.com>
```

Reply:
Paul is BDFL for packaging, so it's implied if he doesn't pick someone. And so far he hasn't been pressed to pick someone, so I think it's premature to specify a delegate.